php/Terminus/Helpers/InputHelper.php

```diff
@@ -376,62 +376,60 @@ public function optional(array $arg_options = [])
    * Input helper that provides interactive menu to select org name
    *
    * @param array $arg_options Elements as follow:
-   *        array  args       The args passed in from argv
-   *        string key        Args key to search for
-   *        string default    Returned if arg and stdin fail in interactive
-   *        array  allow_none True to permit no selection to be an option
+   *        array  args            The args passed in from argv
+   *        string key             Args key to search for
+   *        string default         Returned if arg and stdin fail in interactive
+   *        array  allow_none      True to permit no selection to be an option
+   *        bool   autoselect_solo Automatically selects the only member
    * @return string ID of selected organization
    * @throws TerminusException
   */
     public function orgId(array $arg_options = [])
     {
         $default_options = [
-        'args'       => [],
-        'key'        => 'org',
-        'default'    => null,
-        'allow_none' => true,
+            'args'            => [],
```

Comment:
This change prevents the only org being present from being automatically chosen.